src/frontend/src/pages/CalendarPage/DesignReviewDetailPage/UserAvailabilitesView.tsx

```diff
@@ -85,19 +92,22 @@ const UserAvailabilites: React.FC<UserAvailabilitiesProps> = ({
                 width: '10vw'
               }}
             >
-              Unvailable
+              Unavailable
             </Typography>
-            {currentUnavailableUsers.map((user) => (
-              <Typography style={{ textAlign: 'center', fontSize }}>{fullNamePipe(user)}</Typography>
-            ))}
+            <Box sx={{ maxHeight: '350px', overflowY: 'scroll' }}>
+              {currentUnavailableUsers.map((user) => (
+                <Typography style={{ textAlign: 'center', fontSize: '1em' }}>{fullNamePipe(user)}</Typography>
+              ))}
+            </Box>
           </Box>
         </Box>
         <Box
           style={{
             marginTop: 'auto',
             display: 'flex',
             justifyContent: 'flex-end',
-            gap: '10px'
+            gap: '10px',
+            overflow: 'scroll'
```

Comment:
what are all the code changes in this file for, it's doing some weird stuff to the scrollbars:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/f8b68df2-a662-4673-a3a4-6cdc3b1e5124)

Reply:
i think what i was trying to do was to have the scrollbar always show even when not scrolling, but i dont think we found a way to do this. i changed scroll to auto.